server/mqtt.go

```diff
@@ -1873,11 +1875,11 @@ func (as *mqttAccountSessionManager) sendJSAPIrequests(s *Server, c *client, acc
 // or 0 if the record was added instead of updated.
 //
 // Lock not held on entry.
-func (as *mqttAccountSessionManager) handleRetainedMsg(key string, rm *mqttRetainedMsg) uint64 {
+func (as *mqttAccountSessionManager) handleRetainedMsg(key string, rm *mqttRetainedMsgRef) uint64 {
```

Comment:
It looks like with this (and previous changes you made turning it into a KV mode), we don't need to return the "old" sequence number, since we don't need to delete the message anymore. So should we change the signature?

Reply:
I think there's one case where we still do: when handling an empty MQTT message, which is "equivalent" to a delete?

Reply:
No, I searched for all references to this function in your branch and we no longer use the returned value. The one you are referring to is called `handleRetainedMsgDel()` (notice the `Del` at the end).